CMakeLists.txt

```diff
@@ -32,6 +32,14 @@ list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/cmake/")
 
 include(CTest)
 
+option(USE_SYSTEM_DATE "\
+    Use find_package to find installed HowardHinnant's \
+    date library instead of fetching it from github" OFF
+)
+
+if(USE_SYSTEM_DATE)
+    find_package(date REQUIRED)
+endif()
 
 ### Dependencies 
 add_subdirectory(dependencies)
```

Comment:
```suggestion

### Dependencies 
if(USE_SYSTEM_DATE)
    find_package(date REQUIRED)
else()
    add_subdirectory(dependencies)
endif()
```

Reply:
Just found out there is suggest changes button :-)  But not tested, please check if it works